azure/notebooks/Azure-MNMG-RF.ipynb

```diff
@@ -8,7 +8,7 @@
     "\n",
```

Comment:
<p>Might be worth mentioning why you are multiplying by 2 here since this is VM specific, or just replace this with <code>npartitions=len(client.has_what().keys())</code> if you don't want to open a can of worms here :D</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cloud-ml-examples/pull/133/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>